test/controllers/to_do_lists_controller_test.rb

```diff
@@ -31,6 +31,13 @@ def setup
     assert_not_nil assigns(:list)
   end
 
+  test "should search through the lists" do
+    xhr :post, :search, search: 'is:todo author:vasyll@tada.com Read'
+
+    assert_response :success
```

Comment:
@vlasiak Please correct into POST please as it might be confusing (the test passes though).